gradio/mix.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, *interfaces: gradio.Interface, **options):
         Returns:
             an Interface object comparing the given models
         """
-        outputs: List[IOComponent] = []
+        outputs: List[Component] = []
```

Comment:
It does have to be an IOComponent, right?

Reply:
Yeah I'm actually going to just remove this type hint